quant.module

```diff
@@ -807,43 +819,51 @@ function quant_log_skip_seed($url, $context) {
     return FALSE;
   }
 
+  if (empty($context['type'])) {
+    return;
+  }
 
-  // @todo check other data like the hash?
-  if (isset($context['type'])) {
-    // Handle entities.
-    if (isset($context['id'])) {
-      $entities = entity_load($context['type'], [$context['id']]);
-      if (empty($entities)) {
-        // Something is wrong with the type or id.
-        quant_log(t('Could not load entity %type (%id)', ['%type' => $context['type'], '%id' => $context['id']]));
-        return TRUE;
+  // Handle entities.
+  if (isset($context['id'])) {
+    $entities = entity_load($context['type'], [$context['id']]);
+    if (empty($entities)) {
+      // Something is wrong with the type or id.
+      quant_log(t('Could not load entity %type (%id)', ['%type' => $context['type'], '%id' => $context['id']]));
+      return TRUE;
+    }
+    $entity = reset($entities);
+    $changed = isset($entity->changed) ? $entity->changed : (isset($entity->created) ? $entity->created : (isset($entity->timestamp) ? $entity->timestamp : 0));
+
+    // Check the quant_log data for this entity.
+    $logs = db_query('SELECT * FROM {quant_log} WHERE entity_id = :entity_id AND type = :type', array(':entity_id' => $context['id'], ':type' => $context['type']));
+    foreach ($logs as $log) {
+      // Reprocess if status or log message aren't as expected.
+      if ($log->status != '200' || $log->message != 'Successful.') {
+        return FALSE;
       }
-      $entity = reset($entities);
-      $changed = isset($entity->changed) ? $entity->changed : (isset($entity->created) ? $entity->created : (isset($entity->timestamp) ? $entity->timestamp : 0));
-
-      // Check the quant_log data for this entity.
-      $logs = db_query('SELECT * FROM {quant_log} WHERE entity_id = :entity_id AND type = :type', array(':entity_id' => $context['id'], ':type' => $context['type']));
-      foreach ($logs as $log) {
-        // Reprocess if status or log message aren't as expected.
-        if ($log->status != '200' || $log->message != 'Successful.') {
-          return FALSE;
-        }
-        // Check the log date against the entity.
-        if ($changed != 0 && $changed < $log->updated) {
-          return TRUE;
-        }
+      // Check the log date against the entity.
+      if ($changed != 0 && $changed < $log->updated) {
+        return TRUE;
       }
     }
-    elseif ($context['type'] === 'file') {
+  }
+
+  switch ($context['type']) {
+    case 'file':
+    case 'file_attachment':
+    case 'theme':
       // Check the log md5 against the context md5.
       if (isset($context['location']) && isset($context['md5'])) {
-        $log_md5 = db_query('SELECT md5 FROM {quant_log} WHERE url = :url AND type = :type', array(':url' => $url, ':type' => 'file'))->fetchField();
+        $log_md5 = db_query('SELECT md5 FROM {quant_log} WHERE url like :url AND type = :type', array(
+          ':url' => '%' . db_like($url),
```

Comment:
`$url` for theme assets is sans host (eg. `/themes/bartik/style.css` and in the DB the log item is stored with `http://ngin:8080/themes/bartik/styles.css` — so the like query here _should_ help us be a little loser on the md5 lookup.

We still hash the location and confirm, so likelihood this will result in a false positive should be fairly low.